internal/autoupdate/pages/static/list.css

```diff
@@ -16,49 +15,80 @@ a {
 }
 
 body {
-  font-family:  ui-monospace,SFMono-Regular,SF Mono,Consolas,Liberation Mono,Menlo,monospace;
+  font-family: ui-monospace, SFMono-Regular, "SF Mono", Consolas, "Liberation Mono", Menlo, monospace;
   line-height: 1.2;
 }
 
-table[class="queue_table"] {
+table#queue-table {
   border: 1px solid #D9D7CE;
-  width: 80ch;
+  width: 100%;
+  table-layout: fixed;
   font-size: 1.1em;
   background-color: #eeeeee;
   padding: 5px;
   text-align: left;
 }
 
-table[class="queue_table"] th {
+table#queue-table th {
   border: 10px solid black;
   background-color: black;
   color: white;
 }
 
+table#queue-table tr:nth-child(odd) {
+}
 
-table tr[class="tr_active_prs"] {
+table#queue-table tr.active-prs {
   background-color: #eeeeee;
 }
 
-table tr[class="tr_suspended_prs"] {
-  background-color: #cccccc ;
+table#queue-table tr.suspended-prs {
+  background-color: #cccccc;
 }
 
-table td[id="td_status_updating"] {
-  color: #0eb200;
+table#queue-table td {
+  padding: 5px;
+  overflow: hidden;
+}
+
+table#queue-table th.rank, table#queue-table td.rank {
+  width: 50px;
+}
+
+table#queue-table th.pr, table#queue-table td.pr {
+  width: auto;
+  word-break: break-all;
+}
+
+table#queue-table th.author, table#queue-table td.author {
+  width: 200px;
+}
+
+table#queue-table th.enqueue-ts, table#queue-table td.enqueue-ts {
+  width: 250px;
+}
+
+table#queue-table th.status, table#queue-table td.status {
+  width: 100px;
```

Comment:
Shouldn't we use relative units instead of px for containers holding text, to scale smoothly on different resolutions?

Reply:
btw, i think we could potentially shorten the "Ready for updates since" column by using a different column label and omitting the timezone from the ts, eg:

```
Ready since (UTC)
----------
2023-02-20 11:37:13
```

Reply:
Sounds good, we would have to ensure that all timestamps are always in UTC.
I don't remember out of my head how it currentlya works. I guess it depends on the OS timezone currently.

It would be even better to show them in localtime. We probably would need use javascript for that though.
Alternatively we could add a config option for the used timezone and then always convert it to that one. That should be sufficient for the current needs.

Reply:
We could also show the duration since page refresh instead of the absolute time. :thinking: